iree/compiler/Api/python/test/transforms/ireec/compile_sample_module.py

```diff
@@ -25,40 +26,28 @@
 from iree.compiler.dialects import tosa
 from iree.compiler.dialects import vector
 
-from iree.compiler.transforms import ireec
-
 # Test the compiler API.
 with ir.Context() as ctx:
-  chlo.register_chlo_dialect(ctx)
-  mhlo.register_mhlo_dialect(ctx)
-  iree_input.register_dialect(ctx)
+  ireec.register_all_dialects(ctx)
 
   input_module = ir.Module.parse(r"""
     builtin.module  {
-      builtin.func @fabs(%arg0: tensor<1x4xf32>, %arg1: tensor<4x1xf32>) -> tensor<4x4xf32> {
+      func.func @fabs(%arg0: tensor<1x4xf32>, %arg1: tensor<4x1xf32>) -> tensor<4x4xf32> {
         %0 = chlo.broadcast_add %arg0, %arg1 : (tensor<1x4xf32>, tensor<4x1xf32>) -> tensor<4x4xf32>
         %1 = "mhlo.abs"(%0) : (tensor<4x4xf32>) -> tensor<4x4xf32>
         return %1 : tensor<4x4xf32>
       }
     }
   """)
 
-  options = ireec.CompilerOptions("--iree-input-type=mhlo",
-                                  "--iree-hal-target-backends=cpu")
+  options = ireec.CompilerOptions("--iree-hal-target-backends=vmvx")
```

Comment:
Wouldn't cpu backend here exercise more components and be better smoke test?

Reply:
Indeed.